.github/workflows/pythonpackage.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10.0-alpha.4"]
```

Comment:
I think there is a beta version now

Reply:
Yep unfortunately it doesn't seem to have been added to Github's workflows yet 🙂 Tried the beta version [here](https://github.com/willmcgugan/rich/runs/2647076941?check_suite_focus=true) and the workflow raises an error saying `Version 3.10.0b1 with arch x64 not found`. Maybe they have to add them in manually and there's a lag?

For context on the PR, I was hoping to verify that the [install poetry](https://github.com/snok/install-poetry) action now worked well with 3.10, related to [this issue](https://github.com/willmcgugan/rich/issues/1216), and the action looks to install OK in [this run](https://github.com/willmcgugan/rich/runs/2647562295?check_suite_focus=true), but there seems to also be a resolver issue. It does looks fixable though, so I think I'll try and tinker with it a little bit to see if I can get it to work 👍

Reply:
Should the beta version not being given as "3.10.0-beta.1" ? It looks like in the workflow it is given as "3.10.0-b1"

I found the list of Python versions [here](https://github.com/actions/python-versions/blob/main/versions-manifest.json)

Reply:
Nice 👍 Added to the matrix, you're absolutely right